benefits/enrollment/templates/enrollment/success--sacrt-light-rail.html

```diff
@@ -0,0 +1,21 @@
+{% extends "enrollment/success.html" %}
```

Comment:
Is there a difference between `success--sacrt-light-rail.html` and `success--sacrt.html`?

We aren't yet live with SacRT and I think by the time we are, it will cover more than just light rail. The user has no way of choosing "light rail only" benefits, so I don't think this template is needed?

Reply:
I created `success--sacrt-light-rail.html` for the eventual case when SacRT offers a benefit similar to an agency card but I think you're right, this template is not needed (and it's not even referenced in `local_fixtures.json` now so it's probably just confusing) so I'll remove it. By the time we go live with SacRT we'll revise them 👍

Reply:
Yeah I think the other thing to keep in mind (a "philosophy" if you will) is that we try to avoid creating code for situations we think might come up in the future.

As it stands now, we haven't talked with SacRT about an agency card flow (not even sure if they have an agency card?) so better to come back to it later if we do end up needing it.

Reply:
I agree, it does keep the code more organized and it's easier to follow 👍